Testing/SystemTests/tests/analysis/SaveNexusTest.py

```diff
@@ -0,0 +1,102 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
It is very thorough of you to write this so 👍 for attempting this, but I don't think this should hold up the PR unless it is demonstrating real errors that we need to fix. We can add the system test as a follow up issue providing that it is not demonstrating something that needs to be fixed now relating to the `SaveNexusGeometry` functionality.